lib/consts.dart

```diff
@@ -22,6 +22,36 @@ const kNetworkEndian = Endian.little;
 const kDbName = "forest_park_reports";
 const kImageDirectory = "images";
 
+/// An assortment of calculated values regarding the panel  
+class PanelValues {
+  // The fraction of the total window height to which the panel should open
```

Comment:
nit: These can be made into doc comments as well, even though they're private (helps for ide tooltip support).